lib/blueprinter/view.rb

```diff
@@ -14,10 +14,6 @@ def initialize(name, fields: {}, included_view_names: [], excluded_view_names: [
       @sort_by_definition = Blueprinter.configuration.sort_fields_by.eql?(:definition)
     end
 
-    def transformers
-      view_transformers.empty? ? Blueprinter.configuration.default_transformers : view_transformers
-    end
```

Comment:
Why remove this rather than update? Was it unused?

Reply:
Retained as it is.